types/marked/tslint.json

```diff
@@ -1,3 +1,7 @@
 {
-    "extends": "@definitelytyped/dtslint/dt.json"
+    "extends": "@definitelytyped/dtslint/dt.json",
+    "rules": {
+        "void-return": false,
+        "no-void-expression": false
```

Comment:
I had to relax these rules in order to assert that the overloads with callbacks would actually return `void`.